src/snowflake/snowpark/dataframe.py

```diff
@@ -1414,6 +1414,11 @@ def select(
                 names.append(e._named())
                 if _emit_ast and _ast_stmt is None:
                     ast_cols.append(e._ast)
+            elif isinstance(e, (list, tuple, set)):
+                for sub_e in e:
+                    names.append(sub_e._named())
+                    if _emit_ast and _ast_stmt is None:
+                        ast_cols.append(sub_e._ast)
```

Comment:
how does this change affect this new funcs?

Reply:
json_tuple will have to return a list of columns, while our select function currently does not support this form of call:
df.select(col1, [col2,col3]), this is meant to support it